### DISCUSSION TITLE
Export PNG image from Schema Editor

### DISCUSSION THREAD

**Discussion by henriquek3** [TiagoSilvaPereira/vemto-issues#194] [Ideas & Suggestions]
Is it possible to implement the option to export a png file from the Schema Editor tab to share the project's relationships as images, as in the example below?
![image](https://user-images.githubusercontent.com/17914565/125295517-4c4d5980-e2f3-11eb-9ae4-563ea8b51c63.png)

**Comment by TiagoSilvaPereira** [comment]
Hi @henriquek3 thank you for your suggestion... as it is not a bug, I'm converting it to a discussion. Best regards
